src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GiftOptionsType.php

```diff
@@ -77,7 +92,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('enable_gift_wrapping', SwitchType::class, [
                 'required' => false,
                 'label' => $this->trans('Offer gift wrapping', 'Admin.Shopparameters.Feature'),
-                'help' => $this->trans('Suggest gift-wrapping to customers.', 'Admin.Shopparameters.Help'),
+                'help' => $this->trans('Remember to regenerate email templates after enabling or disabling this option in [1]Design > Email theme[/1].',
```

Comment:
```suggestion
                'help' => $this->trans('Remember to regenerate email templates in [1]Design > Email theme[/1] after enabling or disabling this feature.',
```

Otherwise, it looks like the feature can be enabled/disabled in Design>Email theme